src/main/scala/com/microsoft/hyperspace/index/HyperSpaceIndex.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index
+
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonIgnoreProperties, JsonSubTypes, JsonTypeInfo, JsonValue}
+import com.fasterxml.jackson.annotation.JsonSubTypes.Type
+
+object HyperSpaceIndex {
```

Comment:
Ditto. Seems to have the wrong casing. It should be `Hyperspace`, only one capital letter.